resources/bedrock-flow-alias.go

```diff
@@ -0,0 +1,125 @@
+package resources
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/bedrockagent"
+
+	"github.com/ekristen/libnuke/pkg/registry"
+	"github.com/ekristen/libnuke/pkg/resource"
+	"github.com/ekristen/libnuke/pkg/types"
+
+	"github.com/ekristen/aws-nuke/v3/pkg/nuke"
+)
+
+const BedrockFlowAliasResource = "BedrockFlowAlias"
+
+func init() {
+	registry.Register(&registry.Registration{
+		Name:   BedrockFlowAliasResource,
+		Scope:  nuke.Account,
+		Lister: &BedrockFlowAliasLister{},
```

Comment:
Please add `Resource: &BedrockFlowAlias{}` -- this allows for automatic documentation.